examples/experimental/dagster-airlift/examples/kitchen-sink/kitchen_sink_tests/integration_tests/test_e2e_mapped.py

```diff
@@ -18,7 +18,7 @@ def poll_for_materialization(
     asset_key: AssetKey,
 ) -> EventLogEntry:
     start_time = get_current_datetime()
-    while get_current_datetime() - start_time < timedelta(seconds=30):
+    while get_current_datetime() - start_time < timedelta(seconds=300000):
```

Comment:
assuming this was necessary for testing but let's go back to a reasonable number here

Reply:
yes good catch thanks